<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>numerical_methods_4</class>
 <widget class="QMainWindow" name="numerical_methods_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>numerical_methods_4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="derivacion">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>DERIVACIÓN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="e_gausiana">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ELIMINACIÓN GAUSSIANA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="euler">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EULER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="g_seidel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GAUSS-SEIDEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="integ">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INTEGRACIÓN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="interplacion">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>INTERPOLACION DE LAGRANGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="n_r">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NEWTON-RAPHSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="punt_fijo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PUNTO.FIJO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_lineal">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>REGRESIÓN LINEAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="r_l_m">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>REGRESIÓN  LINEAL MULTIPLE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runge">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RUNGE-KUTTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Miriam Fixed</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numerical Methods</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
